src_plugins/ppx_deriving_eq.cppo.ml

```diff
@@ -148,7 +148,8 @@ let str_of_type ~options ~path ({ ptype_loc = loc } as type_decl) =
     | Ptype_abstract, Some manifest -> expr_of_typ quoter manifest
     | Ptype_variant constrs, _ ->
       let cases =
-        (constrs |> List.map (fun { pcd_name = { txt = name }; pcd_args } ->
+        (constrs |> List.map (fun { pcd_name = { txt = name }; pcd_args; pcd_loc } ->
+          (* with_default_loc pcd_loc @@ fun () -> *)
```

Comment:
That was a debug thing, of course (to observe the previous state), will force-push. Sorry!